src/vss_tools/vspec/vssexporters/vss2samm/helpers/fileHelper.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2024 Contributors to COVESA
+#
+# This program and the accompanying materials are made available under the
+# terms of the Mozilla Public License 2.0 which is available at
+# https://www.mozilla.org/en-US/MPL/2.0/
+#
+# SPDX-License-Identifier: MPL-2.0
+
+import os
+from rdflib import Graph
+from vss_tools import log
+from .. config import config as cnfg
```

Comment:
looks a bit strange to import it as a name that spares only 2 chars. The common abbreviation that people use is probably `cfg`? But I might be biased here